plugin/webhook/webhook.go

```diff
@@ -34,13 +34,32 @@ const (
 	WebhookError Level = "ERROR"
 )
 
+// Issue object of issue
+type Issue struct {
+	ID          int    `json:"id"`
+	Name        string `json:"name"`
+	Status      string `json:"status"`
+	Type        string `json:"type"`
+	Description string `json:"description"`
+}
+
+// Project object of project
+type Project struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
 // Context is the context of webhook.
 type Context struct {
 	URL          string
+	ActivityType string
```

Comment:
nit: let's put composite types like "Issue", "Project" to the last and use this exact order wherever the instance is created.